Tools/MiniBrowser/mac/BrowserWindowController.m

```diff
@@ -324,8 +325,10 @@ - (NSImage *)sharingService:(NSSharingService *)sharingService transitionImageFo
     NSRect contentFrame = [self.window convertRectToScreen:self.mainContentView.bounds];
 
     CGRect frame = coreGraphicsScreenRectForAppKitScreenRect(contentFrame);
+    ALLOW_DEPRECATED_DECLARATIONS_BEGIN
     CGImageRef imageRef = CGWindowListCreateImage(frame, kCGWindowListOptionIncludingWindow, (CGWindowID)[self.window windowNumber], kCGWindowImageBoundsIgnoreFraming);
-    
+    ALLOW_DEPRECATED_DECLARATIONS_END
```

Comment:
Not sure if we should import WTF headers in Minibrowser. This would be the first instance.

Can we just:

```
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
CGImageRef imageRef = CGWindowListCreateImage(frame, kCGWindowListOptionIncludingWindow, (CGWindowID)[self.window windowNumber], kCGWindowImageBoundsIgnoreFraming);
#pragma GCC diagnostic pop
```